man/man8/zfs-mount.8

```diff
@@ -83,6 +83,8 @@ for more information.
 .It Fl a
 Mount all available ZFS file systems.
 Invoked automatically as part of the boot process if configured.
+.It Fl r
+Mount the specified filesystem along with all of it's children.
 .It Ar filesystem
 Mount the specified filesystem.
```

Comment:
Lets document that code allows multiple filesystems specification:
```suggestion
.It Fl r
Mount specified filesystems along with all their children.
.It Ar filesystem
Mount specified filesystems.
```

Reply:
Updated to `Mount specified filesystems along with all their children.`. `zfs mount` without any flag only accepts one dataset at a time, so the next line `Mount specified filesystems.` would be inaccurate.